README.md

```diff
@@ -3,19 +3,238 @@
 The Open Policy Agent (OPA) is an open source, general-purpose policy engine
 that enables unified, context-aware policy enforcement across the entire stack.
 
-## Want to learn about OPA?
+## Want to learn more about OPA?
 
 - See [openpolicyagent.org](http://www.openpolicyagent.org) to get started.
 - See [blog.openpolicyagent.org](https://blog.openpolicyagent.org) for blog posts about OPA and policy.
 - Join the conversation on [Slack](http://slack.openpolicyagent.org).
 
 ## Want to get OPA?
 
-See [GitHub releases](https://github.com/open-policy-agent/opa/releases) for binary releases.
+- See [Docker Hub](https://hub.docker.com/r/openpolicyagent/opa/tags/) for Docker images.
+- See [GitHub releases](https://github.com/open-policy-agent/opa/releases) for binary releases and changelogs.
 
-## Want to build OPA?
+## Want to integrate OPA?
 
-See [DEVELOPMENT.md](./docs/devel/DEVELOPMENT.md) for development environment setup and build instructions.
+* See
+  [![GoDoc](https://godoc.org/github.com/open-policy-agent/opa?status.svg)](https://godoc.org/github.com/open-policy-agent/opa/rego)
+  to integrate OPA into services written in Go.
+
+## Want to contribute to OPA?
+
+* See [DEVELOPMENT.md](./docs/devel/DEVELOPMENT.md) to build and test OPA itself.
+* See [CONTRIBUTING.md](./CONTRIBUTING.md) to get started.
+
+## How does OPA work?
+
+OPA gives you a high-level declarative language to author and enforce policies
+across your stack. With OPA, you define rules that govern how your system
+should behave and then you offload policy decisions from your service to OPA.
+
+The examples below show how you can use OPA to enable policy-based control in
+any system, at any layer. For concrete examples of how to integrate OPA with
+existing systems like
+[Kubernetes](http://www.openpolicyagent.org/docs/kubernetes-admission-control.html),
+[Terraform](http://www.openpolicyagent.org/docs/terraform.html),
+[Docker](http://www.openpolicyagent.org/docs/docker-authorization.html),
+[SSH](http://www.openpolicyagent.org/docs/ssh-and-sudo-authorization.html), and
+more, see [openpolicyagent.org](http://www.openpolicyagent.org).
+
+### Example: API Authorization
+
+This example shows how you can enforce access controls over salary information
+served by a simple HTTP API. In this example, users are allowed to access their
+own salary as well as the salary of anyone who reports to them.
+
```

Comment:
Probably also worth a sentence that says we're showing queries as they are evaluated in the REPL, which was designed to make it easy to test the policies you write and the queries you want to run before actually integrating them into your system.